server/model/promo.js

```diff
@@ -0,0 +1,11 @@
+import {Record} from 'immutable';
+
+export const Promo = Record({
+  modifiers: null,
+  url: null,
+  image: null,
+  meta: null,
+  title: null,
+  copy: null,
+  type: null
+});
```

Comment:
We don't need to populate the whole Article, just a few fields.
[I had opened a PR](https://github.com/wellcometrust/wellcomecollection.org/pull/383) as to how we might start doing that.